Repositories/ProjectRepository.cs

```diff
@@ -281,6 +287,29 @@ public override void Update(Project entity)
             DbSet.Update(entity);
         }
 
+        /// <summary>
+        ///     Get the user projects.
+        /// </summary>
+        /// <param name="userId">The id of the user whoms projects need to be retrieved</param>
+        /// <returns>A enumerable of the users projects</returns>
+        public async Task<IEnumerable<Project>> GetUserProjects(int userId)
+        {
+            IEnumerable<Project> projects = await GetDbSet<Project>()
+                   .Include(p => p.Collaborators)
+                   .Include(p => p.ProjectIcon)
+                   .Where(p => p.UserId == userId)
+                   .ToListAsync();
+
+            foreach(Project project in projects)
+            {
+                project.Collaborators = await GetDbSet<Collaborator>()
+                                              .Where(p => p.ProjectId == project.Id)
+                                              .ToListAsync();
+            }
```

Comment:
Collaborators are already included on line 298, right? I think these lines can be removed.

Reply:
> Great work!
> 
> I found some redundant code which slows down our application. You can remove those the lines. I already tested it and it works.

And don't forget to change the changelog